android/app/src/main/AndroidManifest.xml

```diff
@@ -7,7 +7,6 @@
       android:name=".MainApplication"
       android:label="@string/app_name"
       android:icon="@mipmap/ic_launcher"
-      android:roundIcon="@mipmap/ic_launcher_round"
```

Comment:
I removed the roundIcon as it is not needed and in this case we didn't have it in the design.